annotations/docker-annotations/src/main/java/io/ap4k/docker/decorator/ApplyRegistryToImageDecorator.java

```diff
@@ -21,24 +21,28 @@
 import io.ap4k.kubernetes.decorator.ApplyImageDecorator;
 import io.ap4k.kubernetes.decorator.Decorator;
 import io.ap4k.utils.Images;
+import io.ap4k.Resources;
 
 public class ApplyRegistryToImageDecorator extends Decorator<DeploymentBuilder> {
 
-  private final String name;
-  private final String image;
+  private final String registry;
+  private final Resources resources;
```

Comment:
Does it make sense to use `Coordinates` instead of `Resources`?

Reply:
It felt wrong to pass Resources around while I only needed name, version etc.